virtual_rainforest/models/animals/constants.py

```diff
@@ -0,0 +1,36 @@
+"""The `models.animals.constants` module contains a set of dataclasses containing
+constants" (fitting relationships taken from the literature) required by the broader
+:mod:`~virtual_rainforest.models.animals` module
+"""  # noqa: D205, D415
+
+from dataclasses import dataclass
+
+
+@dataclass
+class DamuthsLaw:
+    """From (Damuth 1987)."""
+
+    exponent: float = -0.75
+    """"""
+    coefficienct: float = 4.23
```

Comment:
Typo `coefficienct` => `coefficient` (and below).

Have you tried this extension? https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker